packages/hash/frontend/src/components/hooks/useUpdateCommentText.ts

```diff
@@ -1,14 +1,15 @@
 import { useMutation } from "@apollo/client";
 import { TextToken } from "@hashintel/hash-shared/graphql/types";
 import { useCallback } from "react";
+import { EntityId } from "@hashintel/hash-subgraph";
 import {
   UpdatePersistedCommentTextMutation,
   UpdatePersistedCommentTextMutationVariables,
 } from "../../graphql/apiTypes.gen";
 import { updatePersistedCommentText } from "../../graphql/queries/comment.queries";
 import { getPersistedPageComments } from "../../graphql/queries/page.queries";
 
-export const useUpdateCommentText = (pageId: string) => {
+export const useUpdateCommentText = (pageId: EntityId) => {
```

Comment:
Continuation of the entity nit: 
```suggestion
export const useUpdateCommentText = (pageEntityId: EntityId) => {
```